File: mobile/src/main/java/com/siliconlabs/bledemo/Browser/ServicesConnectionsCallback.java
package com.siliconlabs.bledemo.Browser;

import com.siliconlabs.bledemo.Bluetooth.BLE.BluetoothDeviceInfo;

public interface ServicesConnectionsCallback {
    void onDisconnectClicked(BluetoothDeviceInfo deviceInfo);

    void onDeviceClicked(BluetoothDeviceInfo device);
}

File: mobile/src/main/java/com/siliconlabs/bledemo/Browser/ServicesConnectionsCallback.kt
package com.siliconlabs.bledemo.Browser

import android.bluetooth.BluetoothDevice
import com.siliconlabs.bledemo.Bluetooth.BLE.BluetoothDeviceInfo

interface ServicesConnectionsCallback {
    fun onDisconnectClicked(deviceInfo: BluetoothDeviceInfo?)
    fun onDeviceClicked(device: BluetoothDevice)
}
